.github/workflows/firebase-merge.yml

```diff
@@ -13,16 +13,15 @@ jobs:
       - uses: actions/checkout@v2
       - run: npm install -g yarn
       - run: yarn
+      - run: cd src
+      - run: npm install firebase-functions@latest firebase-admin@latest --save
```

Comment:
You don't need this step because those libraries are installed with `yarn` command in line 15